### PR TITLE
Report a bug link at bottom of extension edit page goes to 404 error

### LINKED ISSUE #16884: report a bug link at bottom of extension edit page goes to 404 error
### Describe the problem and steps to reproduce it:

when i visit the following page
https://addons.mozilla.org/en-US/developers/addon/revert-site/edit
there is a "Report a bug" link at the bottom of the page
when i click it, it goes to a 404 error page

### What happened?
clicking the "Report a bug" link goes to a 404 error at  https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/AMO/Policy/Contact 

### What did you expect to happen?
it should take me to https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Contact_us

### PR DESCRIPTION
**Pull request by bhushan-borole**
Fixes #16884 

@willdurand

### CODE REVIEW

File: src/olympia/templates/photon-footer.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h4 class="Footer-links-header"><a href="{{ url('home') }}">Add-ons</a></h4>
                 <li><a href="{{ url('devhub.index') }}">Developer Hub</a></li>
                 <li><a href="https://developer.mozilla.org/docs/Mozilla/Add-ons/AMO/Policy">Developer Policies</a></li>
                 <li><a href="https://discourse.mozilla-community.org/c/add-ons">Forum</a></li>
-                <li><a class="Footer-bug-report-link" href="https://developer.mozilla.org/docs/Mozilla/Add-ons/Contact_us">Report a bug</a></li>
+                <li><a class="Footer-bug-report-link" href="https://addons.mozilla.org/en-US/about">Report a bug</a></li>
```
Comment (willdurand) [inline src/olympia/templates/photon-footer.html:17]: Could you please remove the locale from this link?

```
https://addons.mozilla.org/about
```

**Review by willdurand — CHANGES_REQUESTED** [review]
Looks good, thanks. I requested a change to remove the locale from the link because our app is able to set the right locale automatically.

File: src/olympia/templates/photon-footer.html (hunk 1 above)
Comment (bhushan-borole) [reply src/olympia/templates/photon-footer.html:17]: I did a stupid mistake of deleting the fork, I have cloned it again.
Can you please guide me further?

Comment (bhushan-borole) [reply src/olympia/templates/photon-footer.html:17]: Should I create a fresh PR?

Comment (willdurand) [reply src/olympia/templates/photon-footer.html:17]: Not necessarily. You can fetch the current patch locally:

```
curl https://github.com/mozilla/addons-server/pull/17634/commits/f6f962da236c85c0b6bf0508b108f486260c473f.patch | git am -
```

Comment (bhushan-borole) [reply src/olympia/templates/photon-footer.html:17]: I ran the command.
It is showing patch is empty

Comment (willdurand) [reply src/olympia/templates/photon-footer.html:17]: mm, running `git show` should show you this patch. Is it not what you have?

Comment (bhushan-borole) [reply src/olympia/templates/photon-footer.html:17]: It is not showing me this patch, it showing me the patch I did an hour ago to this repo.

Comment (willdurand) [reply src/olympia/templates/photon-footer.html:17]: heh ok, well in this case I suggest you start from scratch given that it is only 1 url to update.

Comment (bhushan-borole) [reply src/olympia/templates/photon-footer.html:17]: Yes sure I will do that xD

**Pull request closed without merging**